src/main/resources/api/swagger.yaml

```diff
@@ -1,74 +1,80 @@
 swagger: '2.0'
 info:
   title: Pazuzu registry
-  description: Generating Dockerfiles by sets of required functions
-  version: "0.0.4"
+  description: Generating Dockerfiles by sets of required functions.
+  version: "0.0.5"
 host: pazuzu.zalando.net
 schemes:
   - http
 produces:
   - application/json
 paths:
-  /api/resolved-features:
-    get:
-      summary: Resolve all features by name
-      description: |
-        Returns all the features with all its dependencies in a single array.
-      tags:
-        - Features
-      parameters:
-        - name: names
-          in: query
-          description: feature names.
-          required: true
-          type: array
-          items:
-            type: string
-      responses:
-        200:
-          description: An array of features
-          schema:
-            type: array
-            items:
-              $ref: '#/definitions/Feature'
-        default:
-          description: Unexpected error
-          schema:
-            $ref: '#/definitions/Error'
   /api/features:
```

Comment:
If /api is your base path, please define it as the base path and leave it out in each resource definition.